tests/test-timber-loader.php

```diff
@@ -302,4 +302,24 @@ public function testTwigLoadsFirstTemplateWhenMultipleLocationsWithSameNamespace
         $str = Timber::compile('@assets/thumb-test.twig');
         $this->assertEquals('<img src="" />', trim($str));
     }
+
+    public function testTwigLoadsFromNotStandardDirectoryInChildTheme()
+    {
+        $this->assertFileExists(WP_CONTENT_DIR . '/themes/timber-test-theme-child-non-standard/style.css');
+        switch_theme('timber-test-theme-child-non-standard');
+        $parent_theme_dir = get_template_directory();
+
+        // Load parent theme functions.php specifically from this directory to fake the caller location.
+        require_once $parent_theme_dir . '/functions.php';
```

Comment:
When placing the 

`Timber::$dirname = ['resources/views'];` in the test directly the tests pass without the fix. This has probably something to do with the caller location. So by requireing the functions.php from the parent theme over here, I mimic the code that led to the bug in the first place. This might be a bit nasty though.